### ISSUE TITLE
Anvilrage Soldier damage bug

### ISSUE THREAD

**Issue by Deekinbc** [slowtorta/turtlewow-bug-tracker#1166]
**Please, provide a clear description what the bug is:**
Anvilrage Soldier attacking for 1000 damage, more than any other mob in the instance. This is far too high.
**Steps to reproduce the behavior with as much detail as possible:**

1. Fight an Anvilrage Soldier
2. 
3. 

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

[NPC: https://database.turtle-wow.org/?npc=](https://database.turtle-wow.org/?npc=8893)
Item: https://database.turtle-wow.org/?item=
Quest: https://database.turtle-wow.org/?quest=
Spell: https://database.turtle-wow.org/?spell=
Object: https://database.turtle-wow.org/?object=

**Expected behavior. Describe how it should work:**
Should be doing half of this at least.

**Comment by Velite12** [comment]
https://youtu.be/vNlqXawaYlo?t=1088

The 5 dwarf packs in the entirety of the Dark Iron Hallway in BRD are overtuned compared to classic and the rest of the instance.

**Comment by Velite12** [comment]
https://i.gyazo.com/thumb/1200/4193374106e68868a2c8eee77858e472-jpg.jpg

**Comment by Velite12** [comment]
`update `creature_template` SET `dmg_min` = 88, `dmg_max` = 119 where `name` = 'Anvilrage Reservist';
update `creature_template` SET `dmg_min` = 187, `dmg_max` = 254 where `name` = 'Anvilrage Overseer';
update `creature_template` SET `dmg_min` = 248, `dmg_max` = 337 where `name` = 'Anvilrage Warden';
update `creature_template` SET `dmg_min` = 262, `dmg_max` = 355 where `name` = 'Anvilrage Guardsman';
update `creature_template` SET `dmg_min` = 338, `dmg_max` = 449 where `name` = 'Anvilrage Captain';
update `creature_template` SET `dmg_min` = 305, `dmg_max` = 412 where `name` = 'Anvilrage Footman';
update `creature_template` SET `dmg_min` = 241, `dmg_max` = 324 where `name` = 'Anvilrage Medic';
update `creature_template` SET `dmg_min` = 273, `dmg_max` = 361 where `name` = 'Anvilrage Officer';
update `creature_template` SET `dmg_min` = 293, `dmg_max` = 397 where `name` = 'Anvilrage Marshal';
update `creature_template` SET `dmg_min` = 279, `dmg_max` = 370 where `name` = 'Anvilrage Soldier';`

**Comment by slowtorta** [comment]
Thanks, @Velite12

**Comment by geojak** [comment]
Yes, thank you all! Now I can finally go brd and hopefully clear that quest without wiping on these packs :) thx!
